UnityProject/Assets/Scripts/Core/Chat/ChatUI.cs

```diff
@@ -347,6 +347,7 @@ public void OnClickSend()
 
 	private void PlayerSendChat(string sendMessage)
 	{
+		sendMessage = sendMessage.Replace("\n", " ").Replace("\r", " ");  // We don't want users to spam chat vertically
```

Comment:
wait what... People could type "\n" in chat and it was parsed as a new line?

Reply:
I could go into notepad and type a word or sentence, hit enter (inserting a return character), and keep doing that then copy and paste all that into the chat window. Then it'd display past its the bounds of the chat bubble and chat box.

Reply:
interesting. Yes, this should do it.